spec/commands/exec_spec.rb

```diff
@@ -139,7 +139,7 @@
 
     bundle! "exec rackup"
 
-    expect(out).to eq("0.9.1")
+    expect(out).to include "0.9.1"
```

Comment:
@colby-swandale Do you have time to review #6949? If we merge that one, we no longer need to update all these specs from `eq` to `include` to take into account that the output stream might include deprecation messages.

Reply:
Ah awesome, that PR is defs a better solution. I will look into getting that merged, but it will take me a bit.

Thanks!

Reply:
Thanks you!

The diff is somewhat big but should be easy to review. The important parts are the changes in `lib/` and `spec/support/helpers.rb`. The rest are boring spec changes.

Once we merge that one, I think we can get clearer PRs both here and #6933 (retargeted to master).